test/test_forest_data.py

```diff
@@ -1,93 +1,38 @@
-        """Load an iris cube"""
-        iris.__version__ = '2.0.0'
-        constraint = iris.Constraint.return_value
-        var_name = 'air_temperature'
-        time_ix = 0
-        field_dict = {
-            'stash_section': None,
-            'stash_item': None
-        }
-        dataset = forest.data.ForestDataset(*self.generic_args(
-            var_lookup = {
-                var_name: field_dict
-            }
-        ))
-        dataset.data[var_name] = {
-        }
-        dataset.basic_cube_load(var_name, time_ix)
-        iris.load_cube.assert_called_once_with(dataset.path_to_load,
-                                               constraint)
-
-    @unittest.mock.patch("forest.data.iris")
-    def test_basic_cube_load_returns_cube(self, iris):
-        iris.__version__ = '2.0.0'
-        var_name = 'air_temperature'
-        time_ix = 0
-        field_dict = {
-            'stash_section': None,
-            'stash_item': None
-        }
-        dataset = forest.data.ForestDataset(*self.generic_args(
-            var_lookup = {
-                var_name: field_dict
+class TestForestDataset(unittest.TestCase):
+    def setUp(self):
+        self.file_name = "file.nc"
+        self.bucket = forest.aws.S3Mount("directory")
+        self.var_name = "var_name"
+        self.var_lookup = {
+            self.var_name: {
+                "stash_section": None,
+                "stash_item": None
             }
-        ))
-        dataset.data[var_name] = {
         }
-        dataset.basic_cube_load(var_name, time_ix)
-        self.assertEqual(dataset.data[var_name][time_ix],
-                         iris.load_cube.return_value)
+        self.dataset = forest.data.ForestDataset(self.file_name,
+                                                 self.bucket,
+                                                 self.var_lookup)
 
-    def generic_args(self, var_lookup=None):
-        config = None
-        file_name = "file_name"
-        s3_base = "s3_base"
-        s3_local_base = "s3_local_base"
-        use_s3_mount = None
-        base_local_path = "base_local_path"
-        do_download = None
-        return (config, file_name, s3_base, s3_local_base, use_s3_mount, base_local_path, do_download, var_lookup)
+    def test_can_be_constructed(self):
+        var_lookup = None
+        forest.data.ForestDataset(self.file_name,
+                                  self.bucket,
+                                  var_lookup)
 
-class TestGetAvailableTimes(unittest.TestCase):
-    def test_get_available_times(self):
-        mock_dataset = unittest.mock.Mock()
-        mock_dataset.get_times.return_value = []
-        datasets = {
-            "key": {
-                "data": mock_dataset
-            }
+    def test_get_var_lookup_mslp(self):
+        config = forest.data.GA6_CONF_ID
+        var_lookup = forest.data.get_var_lookup(config)
+        result = var_lookup['mslp']
+        expect = {
```

Comment:
I cant work out if this test is just checking some config is recorded correctly (not a good test IMO) or some other functionality.  Perhaps we should discuss.